Documentation/ApiOverview/DependencyInjection/Index.rst

```diff
@@ -548,24 +548,30 @@ Symfony: How to Create Service Aliases and Mark Services as Private: SymfonyPriv
 Direct access includes instantiation via :php:`GeneralUtility::makeInstance()`
 with constructor arguments.
 
-This means every class that is directly retrieved using
-:php:`GeneralUtility::makeInstance()` *and* requires dependency injection
-**must** be marked as public. Any other class which requires dependency injection
-and is retrieved by dependency injection itself can be private. Instances of
-:php:`\TYPO3\CMS\Core\SingletonInterface` and Extbase controllers are
+**private**:
+
+*   Any class which requires dependency injection
+    and is retrieved by dependency injection itself can be private.
+
+**public**:
+
+*   Every class that is directly retrieved using :php:`GeneralUtility::makeInstance()` and 
```

Comment:
I dont think the nested list and pseudo headers improve readability...

Reply:
How then can it be improved?

Reply:
One additonal remark - for GU::makeInstance() it is required to avoid providing and constructor arguments to the GU::makeInstance call. Otherwise it will always be created using the new keyword - if registered public or non-public in DI or not.